ci-operator/step-registry/cluster-profiles/cluster-profiles-config.yaml

```
@@ -1860,6 +1869,17 @@ cluster_profiles:
   secret: cluster-secrets-vsphere
 konflux:
   cluster_groups:
-    rh-staging:
+    rh-stg:
     - stone-stg-rh01
     - stone-stage-p01
+konflux:
```

YAMLlint (1.37.1)

[error] duplication of key "konflux" in mapping

(key-duplicates)